.github/workflows/wasm_demos.yaml

```diff
@@ -23,7 +23,18 @@ jobs:
     - name: Gallery WASM build
       run: |
         sed -i "s/#wasm# //" Cargo.toml
-        wasm-pack build --release --target web
+        export SLINT_STYLE=fluent
+        wasm-pack build --release  --outdir pkg/fluent --target -- web
+        export SLINT_STYLE=fluent-light
+        wasm-pack build --release --outdir pkg/fluent-light --target  -- web
+        export SLINT_STYLE=fluent-dark
+        wasm-pack build --release --outdir pkg/fluent-dark --target -- web
+        export SLINT_STYLE=material
+        wasm-pack build --release --outdir pkg/material --target -- web
+        export SLINT_STYLE=material-light
+        wasm-pack build --release --outdir pkg/material-light --target -- web
+        export SLINT_STYLE=material-dark
+        wasm-pack build --release --outdir pkg/material-dark --target -- web
```

Comment:
will this export keep it exported after this? (will it affect the printerdemo and the todo example? i think it would be best to keep these built with the default style)

I think it looks better to keep it in one line  (if it works)
```sh
        SLINT_STYLE=material-dark wasm-pack build --release --outdir pkg/material-dark --target -- web
```

Reply:
done